validator/delphi_validator/errors.py

```diff
@@ -36,7 +36,13 @@ def is_suppressed(self, suppressed_errors):
         errors_to_suppress: Set[Tuple[str]]
             set of (check_name, data_name) tuples to ignore.
         """
-        return (self.check_name, self.data_name) in suppressed_errors
+        if (self.check_name, self.data_name) in suppressed_errors:
+            return True
+        if (self.check_name, "*") in suppressed_errors:
+            return True
+        if ("*", self.data_name) in suppressed_errors:
```

Comment:
These can all go into one "A or B or C" statement, though if you think this is more readable i dont feel strongly

Reply:
Yes I thought with the nesting of parentheses it would be easier to read this way.  Also easier to modify the rules later.